modules/pom.xml

```diff
@@ -35,6 +35,7 @@
         <module>schedulers</module>
         <module>scripting</module>
         <module>scripting-jruby</module>
+        <module>scripting-plugin</module>
```

Comment:
isn't it better to remove the plugin completely and treat it just like any other extension?